### PR TITLE
Added documentation v1.0.0

### PR DESCRIPTION
**Pull request by zelenins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,85 @@
...
+    enabled: false
+```
+
+#### API Gateway Widgets
+
+API Gateway widgets can be globally deactivated by adding an `enabled` flag to the configuration and setting it to false. By default they are enabled.
+
+```yaml
+serverlessDashboard:
+  dynamoDB:
```
Comment (mjovanovic) [inline README.md]: apiGateway

File: README.md [hunk 2]
```diff
@@ -1,2 +1,85 @@
...
+    enabled: false
+```
+
+
+
+## Usage
+Once configured the plugin will run each time you deploy your serverless app and update CloudWatch dashboard to the current structure of the app. 
+
+The message seen after deploy
+![Image of create dashboard](https://i.ibb.co/YQbWYpG/Screenshot-from-2020-03-03-14-32-02.png)
```
Comment (mjovanovic) [inline README.md]: Let's host these files in GitHub

File: README.md [hunk 3]
```diff
@@ -1,2 +1,85 @@
...
+![Image of create dashboard](https://i.ibb.co/9NZMTcV/Screenshot-from-2020-03-03-14-09-33.png)
+
+
+## AWS Dashboard limits
+
+Up to 100 metrics per dashboard widget. Up to 500 metrics per dashboard, across all widgets.
+
+These quotas include all metrics retrieved for use in metric math functions, even if those metrics are not displayed on the graph.
+
+These quotas cannot be changed.
```
Comment (mjovanovic) [inline README.md]: Let's mention the pricing as well?

File: README.md (hunk 1 above)
Comment (zelenins) [reply README.md]: Fixed to apiGateway

File: README.md (hunk 3 above)
Comment (zelenins) [reply README.md]: Pricing header and a link to CloudWatch pricing page added

File: README.md (hunk 2 above)
Comment (zelenins) [reply README.md]: Files are now hosted on github

**Pull request merged**